src/data/polynomial/eval.lean

```diff
@@ -580,15 +580,6 @@ by apply is_ring_hom.of_semiring
 
 instance eval.is_ring_hom {x : R} : is_ring_hom (eval x) := eval₂.is_ring_hom _
 
-lemma eval₂_endomorphism_algebra_map {M : Type w}
```

Comment:
`aeval_endomorphism` cites this, so if you delete this, you need to change the proof of `aeval_endomorphism`.